get_forecast.py

```diff
@@ -86,7 +86,7 @@ def send_slack(slack_url, message):
         return
 
     slack_message = {
-        'text': message
+        'text': '```\n' + message + '```\n'
```

Comment:
How do these differ that we need 2 of them with duplicate code?

Reply:
Please add comment why we duplicate

Reply:
I code this

code_block_format_mssage= '```\n' + message + '```\n'

slack_message = {
         'text':  code_block_format_mssage
}

teams_message = {
         'text':  code_block_format_mssage
}